runtime/manual/tools/repl.md

```diff
@@ -149,8 +149,8 @@ readTextFileSync  readFileSync      readDir           readLink          readAllS
 | Ctrl-Z                | Suspend (Unix only)                                                                              |
 | Ctrl-_                | Undo                                                                                             |
 | Meta-0, 1, ..., -     | Specify the digit to the argument. `–` starts a negative argument.                               |
-| Meta-<                | Move to first entry in history                                                                   |
-| Meta->                | Move to last entry in history                                                                    |
+| Meta &lt;             | Move to first entry in history                                                                   |
+| Meta &gt;             | Move to last entry in history                                                                    |
```

Comment:
I had to remove the `-` here, because `->` creates an arrow, and that would be confusing.